gitlab/client.py

```diff
@@ -441,12 +435,39 @@ def _check_redirects(self, result: requests.Response) -> None:
                 if location and location.startswith("https://"):
                     raise gitlab.exceptions.RedirectError(REDIRECT_MSG)
 
+    def _prepare_send_data(
+        self,
+        files: Dict[str, Any] = None,
+        post_data: Dict[str, Any] = None,
+        raw: Optional[bool] = False,
```

Comment:
Since we have a default value of `False` don't need to support `None` value most likely. Not 100% sure on that.

```suggestion
        raw: bool = False,
```